tools/wpt/browser.py

```diff
@@ -956,7 +956,7 @@ def _chrome_platform_string(self):
         of the Chrome for Testing downloads.
         """
         if self.platform in ("Linux", "Win"):
-            bits = "64" if uname.machine == "x86_64" else "32"
+            bits = "64" if uname.machine in ("x86_64", "AMD64") else "32"
```

Comment:
(Linux emits x86_64 while Windows emits AMD64 here. For future reference, for 64bit ARM processor Linux emits aarch64 while Windows emits ARM64)